### PR TITLE
python 3 conversion

### PR DESCRIPTION
**Pull request by golnazads**

### CODE REVIEW

File: dev-requirements.txt [hunk 1]
```diff
@@ -1,12 +1,17 @@
-pytest==2.8.2
+coveralls==1.8.2
+coverage==4.5.4
+flask-testing==0.8.1
+httmock==1.4.0
+httpretty==0.8.10; python_version < '3.0'
```
Comment (kelockhart) [inline dev-requirements.txt:5]: I _think_ we decided to not make the code backwards compatible with Python 2, so you can remove these.

File: harbour/app.py [hunk 2]
```diff
@@ -3,19 +3,21 @@
 Application factory
 """
 
+from future import standard_library
+standard_library.install_aliases()
```
Comment (kelockhart) [inline harbour/app.py:7]: These two lines are only needed for Python 2 backwards-compatible code, so you can remove them.

File: harbour/client.py [hunk 3]
```diff
@@ -1,3 +1,4 @@
+from builtins import object
```
Comment (kelockhart) [inline harbour/client.py:1]: Also only needed for Python 2-compatible code.

File: harbour/utils.py [hunk 4]
```diff
@@ -19,7 +19,7 @@ def get_post_data(request, types={}):
 
     if types and isinstance(post_data, dict):
         for expected_key in types:
-            if expected_key not in post_data.keys():
+            if expected_key not in list(post_data.keys()):
```
Comment (kelockhart) [inline harbour/utils.py:22]: The syntax `if key in dict.keys()` works ok here in Python 3, without the `list()` - the converter script is really conservative about adding `list()` everywhere, even when it's not needed.

File: harbour/views.py [hunk 5]
```diff
@@ -2,26 +2,27 @@
 """
 Views
 """
+from future import standard_library
+standard_library.install_aliases()
```
Comment (kelockhart) [inline harbour/views.py:6]: These two lines can be removed.

**Review by kelockhart — CHANGES_REQUESTED** [review]
Nearly there! The only main comment is that we'd decided (somewhere in Slack) that we don't need to keep this code backwards compatible with Python 2, so you can remove those lines.

File: config.py [hunk 6]
```diff
@@ -32,6 +32,7 @@
 ADS_TWO_POINT_OH_MIRROR = 'adsabs.harvard.edu'
 
 SQLALCHEMY_DATABASE_URI = ""
+SQLALCHEMY_TRACK_MODIFICATIONS = True
```
[the remark this answers is not in the document]
Comment (golnazads) [reply config.py:35]: somehow I did not get notifications of these comments!
OK, shall change to False.

File: dev-requirements.txt (hunk 1 above)
Comment (golnazads) [reply dev-requirements.txt:5]: OK.

File: harbour/app.py (hunk 2 above)
Comment (golnazads) [reply harbour/app.py:7]: I think these were added automatically when I run futurize. So you are saying I have to go and remove it from all other ones too, lol.

File: harbour/utils.py (hunk 4 above)
Comment (golnazads) [reply harbour/utils.py:22]: I think this is also part of the futurize! OK, thank you.

File: harbour/views.py (hunk 5 above)
Comment (golnazads) [reply harbour/views.py:6]: done.

**Review by kelockhart — APPROVED** [review]
Looks good!

**Pull request merged**